### PR TITLE
DOC-4008-1: Corda 4.9.2 (Ent & Community) patch release

### PR DESCRIPTION
**Pull request by AaronForinton**

### CODE REVIEW

**Review by yarivus — DISMISSED** [review]
LGTM

**Review by nadinequinr3 — DISMISSED** [review]

File: content/en/platform/corda/4.9/community/release-notes.md [hunk 1]
```diff
@@ -16,6 +16,26 @@ title: Release notes
...
+
+* As part of this patch a new Gradle plugin is provided that supports CorDapp development using Gradle 7. This is in addition to the existing Gradle support. The plugin has been uploaded to Artifactory.
+  * Samples demonstrating Gradle 7 usage are available for:
+    * [Kotlin](https://github.com/corda/samples-kotlin/tree/chrisr3-gradle7)
+    * [Java](https://github.com/corda/samples-java/tree/chrisr3-gradle7)
+  * A readme describing the Gradle 7 plugin is also available on [Github](https://github.com/corda/corda-gradle-plugins/tree/release/5.1/cordapp).
+
+### Fixed issues
+
+In this patch release:
```
Comment (AaronForinton) [inline content/en/platform/corda/4.9/community/release-notes.md:31]: These are all bugs -- I've moved two but hesitant to do too much rearranging as already pushing back against pinning issue numbers against them

**Review by yarivus — APPROVED** [review]
LGTM

**Review by yarivus — APPROVED** [review]
LGTM

**Review by yarivus — DISMISSED** [review]
LGTM

**Review by amiegrace12 — DISMISSED** [review]

**Review by yarivus — APPROVED** [review]
LGTM

**Review by amiegrace12 — APPROVED** [review]

**Pull request merged**